evap/contributor/templates/contributor_evaluation_form.html

```diff
@@ -98,9 +98,9 @@ <h5 class="card-title mr-auto">{% trans 'Students' %}</h5>
                 {% if editable %}
                     <button name="operation" value="preview" type="submit" class="btn btn-light">{% trans 'Preview' %}</button>
                     <button name="operation" value="save" type="submit" class="btn btn-primary">{% trans 'Save' %}</button>
-                    {# webtest does not allow submission with value "approve" if no such button exists #}
+                    {# webtest does not allow submission with value "approve" if no such button exists <- still valid? #}
```

Comment:
I have no knowledge about a webtest used in this project. So no, I can't test this.
We can talk next week about it...

Reply:
This is still the case with the current webtest master, so the comment is still valid. I see this rather as an issue on our side though, because we are creating a weird mixup of forms and javascript that adds values to be submitted here.